tasks/fh-build.js

```diff
@@ -63,13 +64,17 @@ module.exports = function(grunt) {
       '!**/*.tar.gz'
     ];
     if (bundle_deps) {
-      _.map(_.keys(grunt.file.readJSON('package.json').dependencies),
-            function(dep) {
-              patterns.push('node_modules/' + dep + '/**');
-              return dep;
-            });
+      exec('npm ls --parseable --production').toString().split('\n').forEach(function(path) {
```

Comment:
`exec` has a limit on the maximum size of the buffer (200kb). Not sure if we ever expet such a large output here, but in that case it would be safer to use `spawn` which streams the output.